apps/system/js/utility_tray.js

```diff
@@ -589,6 +655,42 @@ window.UtilityTray = {
     this.publish('-activated');
   },
 
+  /**
+   * Trigger search from the left half
+   * of the screen if we're LTR And
+   * trigger from the right half
+   * if we're RTL.
+   *
+   * @param  {Touch} touch [description]
+   */
+  _triggerSearchOnTouchEnd: function ut_triggerSearchOnTouchEnd(touch) {
+    var corner;
+
+    if (document.documentElement.dir  == 'rtl') {
+      corner = touch && (touch.target === this.topPanel) &&
+        (touch.pageX > (window.innerWidth / 2));
+    } else {
+      corner = touch && (touch.target === this.topPanel) &&
+        (touch.pageX < (window.innerWidth / 2));
```

Comment:
I didn't touch this logic but both conditions seem to run the same code:

```js
corner = touch && (touch.target === this.topPanel) &&
  (touch.pageX > (window.innerWidth / 2));
```

@albertopq